rust/metaplex/program/src/instruction.rs

```diff
@@ -143,21 +144,22 @@ pub enum MetaplexInstruction {
     ///   5. `[writable]` Vault account
     ///   6. `[writable]` Fraction mint of the vault
     ///   7. `[]` Auction
-    ///   8. `[]` Your BidderMetadata account
-    ///   9. `[signer optional]` Your Bidder account - Only needs to be signer if payer does not own
-    ///   10. `[signer]` Payer
-    ///   11. `[]` Token program
-    ///   12. `[]` Token Vault program
-    ///   13. `[]` Token metadata program
-    ///   14. `[]` Store
-    ///   15. `[]` System
-    ///   16. `[]` Rent sysvar
-    ///   17. `[writable]` Master Metadata account (pda of ['metadata', program id, Printing mint id]) - remember PDA is relative to token metadata program
+    ///   8. `[]` Auction extended
```

Comment:
Yah again, can't do this - you'll break older UIs. Got to put it at the end, or make a new end point.